module/actor/ghoul-actor-sheet.js

```diff
@@ -0,0 +1,186 @@
+/* global Dialog, game, mergeObject */
```

Comment:
Ghouls are mortal with some vampiric powers - and a domitor - so we can inherit the MortalActorSheet